pages/tools/fileconverter.vue

```diff
@@ -0,0 +1,407 @@
+            step="2"
+            @click="currentStep = 2"
+          >
+            <v-row align="center">
+              <v-col cols="auto">
+                Confirm the data type
+              </v-col>
+              <v-col>
+                <v-chip v-if="GeodeObject">
+                  {{ GeodeObject }}
+                </v-chip>
+              </v-col>
+            </v-row>
+          </v-stepper-step>
+
+          <v-stepper-content step="2">
+            <v-row v-if="objects.length">
+              <v-col>
+                <v-row justify="left">
+                  <v-col
+                    v-for="object in objects"
+                    :key="object"
+                    cols="2"
+                    md="2"
+                  >
+                    <v-tooltip bottom>
+                      <template #activator="{ on }">
+                        <v-card
+                          v-ripple
+                          nuxt
+                          class="card ma-2"
+                          hover
+                          elevation="5"
+                          v-on="on"
+                        >
+                          <v-img
+                            :src="
+                              require('@/assets/tools/' +
+                                GeodeObjects[object].image)
+                            "
+                            contain
+                            @click="GetOutputFileExtensions(object)"
+                          />
+                        </v-card>
+                      </template>
+                      <span>{{ GeodeObjects[object].tooltip }}</span>
+                    </v-tooltip>
+                  </v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+            <v-row v-else>
+              <p>
+                This file format isn't supported! Please check the supported
+                file formats documentation for more information
+              </p>
+            </v-row>
+          </v-stepper-content>
+
+          <v-stepper-step
+            :complete="currentStep > 3"
+            step="3"
+            @click="currentStep = 3"
+          >
+            <v-row align="center">
+              <v-col cols="auto">
+                Select file format
+              </v-col>
+              <v-col>
+                <v-chip v-if="extension">
+                  {{ extension }}
+                </v-chip>
+              </v-col>
+            </v-row>
+          </v-stepper-step>
+
+          <v-stepper-content step="3">
+            <v-row v-if="fileExtensions.length" class="flex-column">
+              <v-col>
+                <v-row justify="left">
+                  <v-col
+                    v-for="fileExtension in fileExtensions"
+                    :key="fileExtension"
+                    cols="2"
+                    md="2"
+                  >
+                    <v-card
+                      class="card ma-2"
+                      nuxt
```

Comment:
I think the nuxt tag should be removed (on all the cards in the stepper) since we are not changing page or url